drivers/hmis/app/models/hmis/form/definition.rb

```diff
@@ -57,7 +68,11 @@ class Hmis::Form::Definition < ::GrdaWarehouseBase
       permission: :can_manage_incoming_referrals,
       resolve_as: 'Types::HmisSchema::ReferralRequest',
     },
-    ENROLLMENT: { class_name: 'Hmis::Hud::Enrollment', permission: :can_edit_enrollments, resolve_as: 'Types::HmisSchema::Enrollment' },
+    ENROLLMENT: ENROLLMENT_CONFIG,
+    MOVE_IN_DATE: ENROLLMENT_CONFIG,
+    DATE_OF_ENGAGEMENT: ENROLLMENT_CONFIG,
+    UNIT_ASSIGNMENT: ENROLLMENT_CONFIG,
+    PATH_STATUS: ENROLLMENT_CONFIG,
```

Comment:
These are all basically Enrollment-editing forms, but they need different "roles" so that the frontend can request the correct one.